### ISSUE TITLE
itunes

### ISSUE THREAD

**Issue by despoinamusica39** [alexbcberio/itunes-artwork-finder#5]
hello there...happy new year!could you please complete the project for itunes?for download it from your github?I have no idea from programming!!!I will preciate it so much!!!

**Comment by alexbcberio** [comment]
What do you mean by "complete the project"?
If you want extra features please open a issue of that type.

**Comment by despoinamusica39** [comment]
hello...noooo you dont understand!I mean if you can programmaming for use it....because I have no idea how programming to use it search music!!!

**Comment by alexbcberio** [comment]
Yeah, yeah. If you want me to add some feature to the site you have to open a issue of the type "feature request" filling the asked data.
I will examinate and tell you on there if the thing you want me to do is possible to make

**Comment by despoinamusica39** [comment]
Alex ya veo que eres de Espana!mira.me fascina musica latina y especial pop balada genero!no imaginas cuanto.pero no tengo idea de desarrollo...soy nueva en esto.gatita jaja.te lo ruego puedes funcionarlo y me envies el codigo listo para buscar musica?te lo agradezco mucho!

**Comment by alexbcberio** [comment]
Podrías indicarme que función te gustaría que añadirera a la web? No entiendo que es lo que quieres que haga

**Comment by despoinamusica39** [comment]
primero de todo un placer!Alex refiero que si puedes haras el codigo listo para usarlo....no se como desarrollo tu codigo para abrir en local host

**Comment by alexbcberio** [comment]
Si quieres acceder a la web tienes un enlace en el repositorio, te lo dejo también por aquí: https://alexbcberio.github.io/itunes-artwork-finder/

**Comment by despoinamusica39** [comment]
woowwwwwwww...muchas felicidadesssssssssss...peroooooooooo te lo ruego puedes mostrar ilimitados resultados?salen solamente 200

**Comment by alexbcberio** [comment]
Desafortunadamente no me es posible, para poder utilizar esa funcionalidad Apple te obliga a tener un dispositivo de su marca (iPhone, iPad o iMac). Por ahora no dispongo de ninguno de estos dispositivos

**Comment by despoinamusica39** [comment]
I found a way to query more records. There is a parameter called offset, which is not mentioned in API documentation.

https://itunes.apple.com/search?term=a&offset=25&limit=25

**Comment by despoinamusica39** [comment]
que crees Alex?puedes funcionarlo?seria maravilloso podra buscar ilimitados resultados

**Comment by alexbcberio** [comment]
I'm really confused in what language should I reply 😅

I've done some tests and it works perfectly! I'll try to implement it as this feature was also requested by other user, really appreciate it!!!

Where have you found this? As its not mentioned in the official docs 🤔

**Comment by despoinamusica39** [comment]
jajajajaja aqui amigo!!!en este pagina!!! https://stackoverflow.com/questions/44177089/itunes-search-api-page-number-for-the-query

**Comment by despoinamusica39** [comment]
dime por favorrrrrrrrr...vas funcionarlo mostrar ilimitados?y un favor mas...puedes mostrar la fecha por cada cancion y album?serias muy agradable Alex!!!

**Comment by alexbcberio** [comment]
> dime por favorrrrrrrrr...vas funcionarlo mostrar ilimitados?y un favor mas...puedes mostrar la fecha por cada cancion y album?serias muy agradable Alex!!!

Can you open 2 feature requests for this things explaining this? 
The unlimited results almost sure, for the other thing I will have to think how that extra information would be shown.

**Comment by despoinamusica39** [comment]
woowwwwwwww....eres increible!!!pero dime tal vez puedes hacerlo ahora mismo?si puedes claro!!!

**Comment by alexbcberio** [comment]
The unlimited results shouldn't take me long to do, it should be done in a couple of days, the one of the extra information it will take me much more

**Comment by despoinamusica39** [comment]
woow....eres muy agradable.la verdad muchas gracias.por favor avisame cuando lo haras!un abrazo desde Grecia y Dios que te cuide!

**Comment by alexbcberio** [comment]
You can press on "watch" and you will receive mail notifications whenever I make changes in the repository, I've updated the issue title with the features you requested

**Comment by despoinamusica39** [comment]
siiiiiiiiiiiiiiii ya lo vi....muchisisisisisimas gracias una vez mas!mis saludos hasta la distancia!te cuidas!

**Comment by despoinamusica39** [comment]
buenos diassssss....como estas?

**Comment by alexbcberio** [comment]
> buenos diassssss....como estas?

Hello 😄! 
If you wanna chat its better if we switch to other platform to keep this clean, GitHub does not support private messages between users. You can chat from Twitter (same username as here) better

**Comment by despoinamusica39** [comment]
jajajaja...Si clarooooooo....perdon no lo sabia.jijijiji!por favor una preguntita...porque tengo ganas encontrar musica pop balada tal vez puedes mostrar ilimitados resultados hoy?

**Comment by alexbcberio** [comment]
I'm now working on the project, not sure if I will be able to finish today as I'm changing some internal things too.
I'll let you know for sure when its finished and deployed on GitHub pages

**Comment by despoinamusica39** [comment]
woowwwww....eres increibleeeeeee....ojala que puedas en unas horas o minutos....jajajaja.perdona mi paciencia!mil  gracias por todo!un abrazo hasta la distancia!!!

**Comment by despoinamusica39** [comment]
de veras eres muy buena gente!!!mil gracias por todoooooooooo la verdad.y Alex porque me interesa todas las fechas pero mucho mas 2020  y2021 como desarrollador tal vez puedes hacer un filtro por fecha?creo que es posible!!!mil gracias de antemano.saluditos y un abrazo enorme para ti!!!

**Comment by despoinamusica39** [comment]
de veras eres muy buena gente!!!mil gracias por todoooooooooo la verdad.y Alex porque me interesa todas las fechas pero mucho mas 2020 y2021 como desarrollador tal vez puedes hacer un filtro por fecha?creo que es posible!!!mil gracias de antemano.saluditos y un abrazo enorme para ti!!! (vivaaaaaaaaaa musica pop balada latin y espanol)

**Comment by alexbcberio** [comment]
Update of the status, infinite scrolling is already live. You will now be able to see a infinite number of results.
In order to make this new thing appear you have to load the page, wait some seconds and reload again the page as I forgot to implement a updater on the previous releases 😅 

Regarding the other feature requests you told me:
- Display extra information: there are some things I have to change before starting with this one, its going to take some time.
- Advanced filtering: this feature won't be added in a short time as they are other more important things to do.

**Comment by despoinamusica39** [comment]
woowwwwwwwwwwwww....no tengo palabrasssssssssssssssssssssssssssss...que agradable de tu parte Alex!muchisisisisisisimas gracias con todo mi corazon!espero que muy pronto pones y lo filtro por la fecha!!!seria maravillosooooooooooooooooooooooo!una vez mas muchas gracias y Dios que te cuide siempre a ti y tu familia!!!

**Comment by despoinamusica39** [comment]
Alex te lo ruego si puedes pone manana el filtro por la fecha(dia,mes y ano)es que me interesa mucho los mas recientes pop balada y todas las fechas...pero mucho mas 2021!!!un abrazo!

**Comment by alexbcberio** [comment]
That has very low priority, there are much more important things I have to make first. It also requires more time than the infinite results.

**Comment by despoinamusica39** [comment]
aaaaaaaaa....entiendooo...no te preocupes Alex!crees que hasta el fin de semana quizas lo haras?

**Comment by alexbcberio** [comment]
Uff, one week is nothing. I might need about 3-4 weeks as I also have to go to classes and I got other projects to develop too

**Comment by despoinamusica39** [comment]
ok Alex...cuando puedas amigo.no es urge.toma tu tiempo!por favor avisame cuando lo haras!abrazos desde Grecia y Dios que te cuide!!!

**Comment by alexbcberio** [comment]
Thanks for understanding, I'll update you as I got making progress over here

**Comment by despoinamusica39** [comment]
por supuesto Alex.gracias por todo!!!cuidate!

**Comment by despoinamusica39** [comment]
Alex buenos dias y feliz Sabado!espero que estas bien.amigo al menos puedes mostrar la fecha por cada album y cancion?hasta que pone el filtro por la fecha?te lo ruegoooooooooo!!!

**Comment by alexbcberio** [comment]
As I told you previously, I will require a lot of time to implement the features as I also have to do other things apart this.

Please be patient

**Comment by despoinamusica39** [comment]
muy buenas!disculpame por favor!!!es que te ngo muchas ganas buscar musica 2021...pero tu decides!espero que lo haras muy pronto!un abarzo desde Grecia!
